tenant/config/samples/tenant_v1alpha1_tenant.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: tenant.multitenancy-k8s.io/v1alpha1
```

Comment:
CC @rjbez17 

I though the guidance was literally to use "x-k8s.io", where "x" is the character "x" and not a wildcard. See, for example, https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/919/files, which literally uses "x". 

My assumption was that this was modelled off of the golang.org/x/* packages, which (AIUI) are packages that are not yet considered to be part of the standard library, but are semi-official - kind of like Boost in C++. SIG-endorsed packages would seem to fall into a similar category.

If I'm correct, this would suggest that we call this group something like tenancy.x-k8s.io/v1alpha1

Reply:
yes that is the intent, literal x.